### PR TITLE
capnp 0.5.3.1

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/capnp.rb [hunk 1]
```diff
@@ -26,9 +26,9 @@ def install
     gtest = resource("gtest")
     gtest.verify_download_integrity(gtest.fetch)
     inreplace "src/CMakeLists.txt" do |s|
-      s.gsub! "http://googletest.googlecode.com/files/gtest-1.7.0.zip",
+      s.gsub! "https://github.com/google/googletest/archive/release-1.7.0.zip",
```
Comment (ilovezfs) [inline Formula/capnp.rb]: does 1.8.0 not work?

File: Formula/capnp.rb [hunk 2]
```diff
@@ -39,6 +39,11 @@ def install
   end
 
   test do
-    system "#{bin}/capnp", "--version"
+    file = testpath/"test.capnp"
+    text = "\"Is a happy little duck\""
+
+    file.write Utils.popen_read("#{bin}/capnp id").chomp + ";\n"
+    file.append_lines "const dave :Text = #{text};"
+    assert_match text, shell_output("#{bin}/capnp eval #{file} dave")
```
Comment (ilovezfs) [inline Formula/capnp.rb]: what if it's printing "Is a happy little duck" is invalid syntax... won't that match?

File: Formula/capnp.rb (hunk 1 above)
Comment (DomT4) [inline Formula/capnp.rb]: Unsure, but upstream hardcode in 1.7.0 & I presume that's what they expect other people to test against for now 😓.

Comment (ilovezfs) [inline Formula/capnp.rb]: OK.

File: Formula/capnp.rb (hunk 2 above)
Comment (ilovezfs) [inline Formula/capnp.rb]: I guess that would go to stderr anyway. Also, it looks like we can do `assert_equal` with a chomp.

**Comment by ilovezfs** [comment]
@DomT4 thanks for the PR, and especially for adding a test! :balloon:

File: Formula/capnp.rb (hunk 2 above)
Comment (DomT4) [inline Formula/capnp.rb]: Sorry, missed this one. It's quite specific about its failures. The most common one I could think of being possible here is that someone edits this in future and erases the terminating `;`, which results in this sort of output:
```
==> /usr/local/Cellar/capnp/0.5.3.1/bin/capnp eval /tmp/capnp-test-20170419-86374-169b2ol/test.capnp dave
/tmp/capnp-test-20170419-86374-169b2ol/test.capnp:3:1: error: Parse error.
/tmp/capnp-test-20170419-86374-169b2ol/test.capnp:1:1: error: File does not declare an ID.  I've generated one for you.  Add this line to your file: @0xa0f19a26260eaaef;
/usr/local/Cellar/capnp/0.5.3.1/bin/capnp eval: dave: 'dave' is not defined.
Try '/usr/local/Cellar/capnp/0.5.3.1/bin/capnp eval --help' for more information.
Error: capnp: failed
<0> expected but was
<1>.
```

**Comment by DomT4** [comment]
No worries, happy to help where I can 🙇. Plus it often nudges me to go play with languages/tools I'm not that familiar with, which I find interesting.

**Pull request closed without merging**